solr/CHANGES.txt

```diff
@@ -187,6 +187,8 @@ Bug Fixes
 
 * SOLR-17652: Fix a bug that could cause long leader elections to leave PULL replicas in DOWN state forever. (hossman)
 
+* SOLR-17649: Fix a regression of faceting on multi-valued EnumFieldType, introduced by an earlier 9.x version. (Thomas Wöckinger, David Smiley, Houston Putman, Kevin Risden)
```

Comment:
LOL if I merge this I'll just list yourself; I don't want any hint of credit.  As a reviewer my input was too trivial in this case to justify my name being listed.